tokenize2.js

```diff
@@ -276,7 +276,6 @@
      */
     Tokenize2.prototype.tokenAdd = function(value, text, force){
 
-        value = this.escape(value);
```

Comment:
This was removed because it was trimming some tags out of the value.  With this PR it is no longer required.